Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml

```diff
@@ -661,77 +661,11 @@
         <glowRadius>12</glowRadius>
         <glowColor>(217,217,208,0)</glowColor>
       </li>
-    </comps>
-    <researchPrerequisite>PowerI</researchPrerequisite>
-  </ThingDef>
-
-  <ThingDef ParentName="StandingLampBase">
-    <defName>StandingLamp_Red</defName>
-    <label>standing lamp (red)</label>
```

Comment:
Only if you remove the def in Core. But editing defs in Core is bad. 
The <defName> </defName> is unique, so if it not overriden it will always be there